lib/protoflow-blocks/doc/io/decode_hex.mmd

```diff
@@ -0,0 +1,11 @@
+block-beta
+    columns 7
+    Source space:2 DecodeHex space:2 Sink
+    Source-- "input" -->DecodeHex
+    DecodeHex-- "output" -->Sink
+
+    classDef block height:48px,padding:8px;
+    classDef hidden visibility:none;
+    class DecodeHex block
+    class Source hidden
+    class Sink hidden
```

Comment:
Make sure to newline-terminate all text files

Reply:
done

Reply:
@evrenokcu This one remains problematic (note the round red symbol, GitHub's way of flagging the lack of a newline), although I'm not sure why--are you on Windows, perhaps?

<img width="1155" alt="Screenshot 2024-11-19 at 10 24 24 AM" src="https://github.com/user-attachments/assets/acc4eac1-280b-4a03-bd57-ca652b622d04">

Reply:
I use mac, however I copy-pasted the text from the online mermaid editor, removed and added the last empty line.

Reply:
Looks like the reason for the problem is whitespace on the last line:

<img width="699" alt="Screenshot 2024-11-19 at 5 55 34 PM" src="https://github.com/user-attachments/assets/b0fbcdc7-5fb4-4287-aa24-943556c9cb13">

Reply:
done, I don't see any warnings in github now